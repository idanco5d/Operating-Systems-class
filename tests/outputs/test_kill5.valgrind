==8788== Memcheck, a memory error detector
==8788== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==8788== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==8788== Command: /mnt/c/Users/illus/os/hw/1/smash
==8788== Parent PID: 8782
==8788== 
==8788== 
==8788== FILE DESCRIPTORS: 4 open (3 std) at exit.
==8788== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_kill5.valgrind
==8788==    <inherited from parent>
==8788== 
==8788== 
==8788== HEAP SUMMARY:
==8788==     in use at exit: 0 bytes in 0 blocks
==8788==   total heap usage: 59 allocs, 59 frees, 80,656 bytes allocated
==8788== 
==8788== All heap blocks were freed -- no leaks are possible
==8788== 
==8788== For lists of detected and suppressed errors, rerun with: -s
==8788== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
