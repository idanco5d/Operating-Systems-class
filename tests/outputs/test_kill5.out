smash> smash> smash> signal number 9 was sent to pid 9339
smash> smash> smash: sending SIGKILL signal to 0 jobs:
